profiles/commands.py

```diff
@@ -15,6 +20,38 @@ def _update_name_from_orcid_record(profile: Profile, orcid_record: dict) -> None
                             '{}, {}'.format(family_name, given_name))
 
 
+def _update_affiliations_from_orcid_record(profile: Profile, orcid_record: dict) -> None:
+    orcid_affiliations = orcid_record.get('activities-summary', {}).get('employments', {}).get(
+        'employment-summary', {})
+
+    for index, orcid_affiliation in enumerate(orcid_affiliations):
+        organization = orcid_affiliation['organization']
+        address = organization['address']
+
+        affiliation = Affiliation(str(orcid_affiliation['put-code']),
+                                  countries.get(address['country']), organization['name'],
+                                  date(**{k: int(v) for k, v in
```

Comment:
what a coincidence that `pendulum.create` takes exactly the same arguments as the fields contained in `start-date` or `end-date`? Seems too magic?

Reply:
Yeah, had wondered that in 65e473cfea5708c6de6452bfbf28366360188301...